csm/conf/post_install.py

```diff
@@ -175,6 +185,23 @@ def _configure_service_user(self):
         """
         Setup._update_csm_files("<USER>", self._user)
 
+    def _set_syslog_details(self):
+        # read openldap host and port and set it in csm config file
+        Log.info("Storing rsyslog details")
+        syslog_host = self._fetch_key_value(f"{const.CORTX}>{const.SOFTWARE}> \
+            {const.SYSLOG}>{const.HOST}", "")
+        syslog_port = self._fetch_key_value(f"{const.CORTX}>{const.SOFTWARE}> \
+            {const.SYSLOG}>{const.PORT}", "")
+        # Edit Current Config File.
+        if syslog_host and len(syslog_host) > 1:
+            Log.info("rsyslog host and port details")
+            Conf.set(const.CSM_GLOBAL_INDEX, f"{const.LOG}>{const.SYSLOG_HOST}",
+                    syslog_host)
+            Conf.set(const.CSM_GLOBAL_INDEX, f"{const.LOG}>{const.SYSLOG_PORT}",
+                    syslog_port)
+            self._is_syslog = True
```

Comment:
Set self._is_syslog = False by default... It will be true at this line...
Other wise it will give NoAttribute error ...that _is_syslog not in self... wherever referecned